src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -44,5 +50,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <Target Condition="'$(TargetsWindows)' == 'true'" Name="BuildNetfxRemoteInvoker" AfterTargets="BeforeBuild">
+    <MSBuild Projects="RemoteInvokeNetfx\RemoteInvokeNetfx.csproj" Properties="Configuration=Debug; Platform=$(Platform); OutputPath=$(OutputPath); IntermediateOutputPath=$(IntermediateOutputPath)" />
```

Comment:
This looks super hacky and wrong, although I don't yet fully understand why it's needed. cc: @weshaggard @tarekgh